src/mainboard/emulation/qemu-armv7/fixed-dtfs.dts

```diff
@@ -14,56 +14,56 @@
                 description = "Boot Blob";
                 compatible = "ore-bootblob";
                 offset = <0x0>;
-                size = <0x80000>; // 512KiB
+                size = <0x880000>; // 8.5MiB
                 file = "$(TARGET_DIR)/bootblob.bin";
             };
             area@1 {
                 description = "Fixed DTFS";
                 compatible = "ore-dtfs";
-                offset = <0x80000>;
+                offset = <0x880000>;
```

Comment:
Could omit all of these `offset` entries if you wanted to; IIRC they're auto-computed. I'm not sure that being explicit here actually adds any value to a human reader.

Reply:
Good point, removed the offset.